playwright/tests/screenshots/editor/editor.spec.ts

```diff
@@ -42,7 +42,9 @@ test('extensions', async ({ page }) => {
 
 test('connector-process', async ({ page }) => {
   const editor = await ProcessEditor.openProcess(page, { file: '/processes/market/erp.p.json', waitFor: '.sprotty-graph' });
-  const inscription = await editor.element('start:callSubStart').inscribe();
+  const subStart = editor.element('start:callSubStart');
+  const inscription = await subStart.inscribe();
+  await expect(subStart.locator().getByRole('img').first()).toHaveAttribute('src', /.+user.png/);
```

Comment:
no idea how to make sure this resource is really loaded :thinking: 

maybe you have a hint on Monday @ivy-lli

Reply:
have a look at the extension palette screenshots, there are also custom images which get loaded and I think I implemented a solution for this problem there.